packages/assets-controllers/src/MultichainAssetsRatesController/MultichainAssetsRatesController.ts

```diff
@@ -156,6 +164,13 @@ const metadata = {
   historicalPrices: { persist: false, anonymous: true },
 };
 
+export type OnAssetsConversionResponseType = {
```

Comment:
Can we rename that to something else that isn't ‘OnAssetsConversionResponse` since it's not relevant to that handler anymore ?